docs/AUTHORS.md

```diff
@@ -6,10 +6,16 @@ Marked takes an encompassing approach to its community. As such, you can think o
 
 Users are anyone using Marked in some fashion, without them, there's no reason for us to exist.
 
+|Name           |GiHub handle   |Decision making                                      |Badges of honor (tag for questions) |
+|:--------------|:--------------|:----------------------------------------------------|------------------------------------|
+|Marti Martz    |@Martii        |OpenUserJS.org Co-Owner / Project Active Maintainer |Open source, of course              |
+
```

Comment:
@Martii thanks, we hadn't figured out what to do on these tables. I was thinking it would be different than the others.

Maybe:

|Individual or organization |Website                        | Submitted by                                                                               |
|:-------------------------|:----------------------|:------------------------------------------------------------|
|OpenUserJS.org                |https://openuserjs.org | Marti Martz (@Martii) Co-Owner / Project Active Maintainer|

If you would like to take the game dynamics (badges) to the OpenUserJS project right on!

Reply:
That looks really good... plus it seems narrower for those not using the Wide GitHub Usercript by xthexder *(I try to take these things into account)*.  May put in the `@` symbol to our organization home `@OpenUserJS` since the linkage is in the website.... the `.org` is the project.

Will change momentarily.

Reply:
Oooooh! That @ thing...maybe a different column?? GitHub project - a lot of our users are other open source projects.

Reply:
> maybe a different column??

Depends on how wide you want it. I use the aforementioned  Userscript constantly but not everyone does so pushing the boundaries with more columns will make more word wraps. The reason I put spaces in between the `/` but didn't seem to do much. GFM can be picky sometimes.

We can try it if you like.

Reply:
I'm not too concerned about the width and word wraps from a PO perspective but it's a valid point. I also like the idea of linking directly to the project. Maybe something like this (adding another use to demonstrate the added value to other users:

|Individual or organization |Website                          |GitHub Project                           |Submitted by |
|:------------------------|:------------------------|:------------------------------|:-------------|
|OpenUserJS.org               |https://openuserjs.org   |                                                    |Marti Martz (@Martii) Co-Owner / Project Active Maintainer                       |
|Node Package Manager  |https://www.npmjs.com |https://github.com/npm/npm   |--                    |

See how some users may not have the direct one-to-one you have??

Reply:
Note: The word wraps will potentially look and feel different here in the code review than on display on our site or by GH later.

Reply:
Ughh GH isn't updating your comments real time... totally missed these... will read after phone call...